src/index.js

```diff
@@ -0,0 +1,284 @@
+    // Transform can be any no-arg BigNumber function and should produce a valid Cents value.
+    // e.g. 'ceil', 'round'
+    if (param == null) {
+      param = {};
+    }
+    const {transform} = param;
+    let result = this.toBigNumber().times(scalar);
+    if (transform != null) {
+      result = this._applyBackwardsCompatibleTransform(result, transform);
+    }
+    return new Cents(result);
+  }
+
+  dividedBy(scalar, param) {
+    // Transform can be any no-arg BigNumber function and should produce a valid Cents value.
+    // e.g. 'ceil', 'round'
+    if (param == null) {
+      param = {};
+    }
+    const {transform} = param;
+    let result = this.toBigNumber().dividedBy(scalar);
+    if (transform != null) {
+      result = this._applyBackwardsCompatibleTransform(result, transform);
+    }
+    return new Cents(result);
+  }
+
+  percent(percent, param) {
+    // Is equivalent to @times(percent / 100, {transform})
+    // but avoids (percent / 100) returning too many sig figs for BigNumber.
+    // TODO(serhalp) this is no longer an issue since BigNumber.js@7.0.0:
+    // https://github.com/MikeMcl/bignumber.js/blob/master/CHANGELOG.md#700. Simplify?
+    if (param == null) {
+      param = {transform: 'round'};
+    }
+    const {transform} = param;
+    const scalar = new BigNumber(percent).dividedBy(100).toNumber();
+    return this.times(scalar, {transform});
+  }
+
+  // Handy aliases.
+  is0() {
+    return this.equals(new Cents(0));
+  }
+  isnt0() {
+    return !this.is0();
+  }
+
+  toString() {
+    return `$${new BigNumber(this.toDollars()).toFixed(2)}`;
+  } // always show 2 decimal places
+
+  // BigNumber.js removed `round()`, `ceil()`, and `floor()` in in v6.0.0. Previously this library
+  // allowed magically calling through to underlying BigNumber.js methods. This is a shim to
+  // continue to transparently continue to support the frequently used `round` "transform" without
+  // breaking backwards compatibility.
+  _applyBackwardsCompatibleTransform(bigNumber, transform) {
+    if (transform === 'round') {
+      return bigNumber.integerValue(BigNumber.ROUND_HALF_UP);
+    } else if (transform === 'floor') {
+      return bigNumber.integerValue(BigNumber.ROUND_FLOOR);
+    } else if (transform === 'ceil') {
+      return bigNumber.integerValue(BigNumber.ROUND_CEIL);
+    } else if (typeof bigNumber[transform] !== 'function') {
+      throw new TypeError(
+        `Cannot apply transform '${transform}', is not a supported BigNumber.js method`,
+      );
+    } else {
+      return bigNumber[transform]();
+    }
+  }
+
+  lt(args) {
+    return Reflect.apply(this.lessThan, this, [...args]);
+  }
+
+  lte(args) {
+    return Reflect.apply(this.lessThanOrEqual, this, [...args]);
+  }
+
+  gt(args) {
+    return Reflect.apply(this.greaterThan, this, [...args]);
+  }
+
+  gte(args) {
+    return Reflect.apply(this.greaterThanOrEqual, this, [...args]);
+  }
+
+  static isValid = function (maybeCents) {
```

Comment:
nitpick: can't you just use this regular syntax?
```suggestion
  static isValid(maybeCents) {
```